doc/THIRD_PARTY_LICENSES.md

```diff
@@ -202,3 +202,61 @@ DAMAGES, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM
 OTHER DEALINGS IN THE FONT SOFTWARE.
 ```
+
+
+### tinyfiledialogs
+``` 
+SPDX-License-Identifier: Zlib
+Copyright (c) 2014 - 2024 Guillaume Vareille http://ysengrin.com
+	 ________________________________________________________________
+	|                                                                |
+	| 100% compatible C C++  ->  You can rename this .c file as .cpp |
```

Comment:
ok I see this comment. That's probably why you changed in to `.cxx`  
I think our build system should be able to handle .c files just fine